ga4gh/datamodel/variants.py

```diff
@@ -592,16 +592,19 @@ class HtslibVariantAnnotationSet(HtslibVariantSet):
     Class representing a single variant annotation set backed by a directory of
     indexed VCF or BCF files.
     """
-    compoundIdClass = datamodel.VariantAnnotationSetCompoundId
-
-    def __init__(self, parentContainer, localId, dataDir, backend):
+    def __init__(self, parentContainer, localId, dataDir,
+                 backend, variantSetId):
         super(HtslibVariantAnnotationSet, self).__init__(
             parentContainer, localId, dataDir, backend)
+        self.compoundIdClass = datamodel.VariantAnnotationSetCompoundId
```

Comment:
Hey @jeromekelleher , did you try this change? It is breaking for me and I'm not sure why. It causes a "not enough fields for compound ID" exception.